src/pages/components/DownloadButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Inconsistency with usage of semicolons and no semicolons.
Check other files as well.